app/src/main/java/com/framgia/fdms/screen/producer/vendor/VendorViewModel.java

```diff
@@ -147,10 +175,11 @@ public void onAddProducerClick() {
             Constant.BundleConstant.BUNDLE_DEVICE);
     }
 
-    @IntDef({ACTION_EDIT_VENDOR, ACTION_ADD_VENDOR})
+    @IntDef({ACTION_EDIT_VENDOR, ACTION_ADD_VENDOR, ACTION_DELETE_VENDOR})
     public @interface Action {
         int ACTION_EDIT_VENDOR = 0;
         int ACTION_ADD_VENDOR = 1;
+        int ACTION_DELETE_VENDOR = 2;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.